models/locale.php

```diff
@@ -19,19 +19,22 @@
     $locale = $_GET['locale'];
 }
 
-// get lang files status from langchecker
-$lang_files = Json::fetch(LANG_CHECKER . "?locale={$locale}&json");
+// Get lang files status from langchecker
+$json_data = new Json();
```

Comment:
FYI, since you don't pass any parameter to the constructor, you could omit parenthesis:
```php
$json_data = new Json;
```